product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -4,57 +4,57 @@ title: Maintenance commands through proxies
 
-As a general rule, you should never perform maintenance operations on a cluster's write leader. 
+As a general rule, don't perform maintenance operations on a cluster's write leader. 
 Maintenance operations such as `VACUUM` can be quite disruptive to the smooth running of a busy server and often detrimental to workload performance.
-Therefore it is best to run maintenance commands on any node in a group that isn't the write leader.
-Generally, this requires you connect directly and issue the maintenance commands on the non-write leader nodes.
-But there are situations where this is not possible.
+Therefore, it's best to run maintenance commands on any node in a group that isn't the write leader.
+Generally, this requires you to connect directly and issue the maintenance commands on the non-write-leader nodes.
+But in some situations, this isn't possible.
 
 ## Maintenance and proxies
 
 Proxies, by design, always connect to and send commands to the current write leader.
-This would usually mean that you should not connect via a proxy to perform maintenance.
-PGD clusters nodes are able to present a direct connection for psql and pgd cli clients which can be used for issuing maintenance commands to the server on those nodes. 
+This usually means that you must not connect to a proxy to perform maintenance.
+PGD clusters nodes can present a direct connection for psql and PGD CLI clients that you can use to issue maintenance commands to the server on those nodes. 
 But there are environment in which the PGD cluster is deployed where a proxy is the only way to access the cluster.
 
 For example, in BigAnimal, PGD clusters are locked down such that the only access to the database is through an instance of PGD Proxy. 
-This reduces the footprint of the cluster and makes it more secure but it does require a different way of sending maintenance requests to the cluster’s nodes.
+This configuration reduces the footprint of the cluster and makes it more secure. However, it requires that you use a different way of sending maintenance requests to the cluster's nodes.
 
-The technique outlined here is generally useful for despatching commands to specific nodes without being directly connected to that node’s server.
+The technique outlined here is generally useful for despatching commands to specific nodes without being directly connected to that node's server.
 
 ## Maintenance commands
 
-When we refer to maintenance commands, we are referring to:
+The term *maintenance commands* refers to:
 
 * `VACUUM` 
-* Non-replicated DDL commands (which you may want to manually replicate)
+* Non-replicated DDL commands (which you might want to manually replicate)
 
 
 ## A note on node names
 
-We will be addressing the servers in the cluster by their PGD cluster node names. To get a list of node names in your cluster, use:
+The servers in the cluster are referred to by their PGD cluster node names. To get a list of node names in your cluster, use:
```

Comment:
You seem to have made this passive

Reply:
We don't write in first person, and I was having a hard time revising to eliminate first person without going passive. Feel free to suggest an edit that doesn't use first person.

Reply:
In this article, the servers in the cluster are identified by their PGD cluster node names.

Reply:
```suggestion
In this discussion, the servers in the cluster are identified by their PGD cluster node names. To get a list of node names in your cluster, use:
```
What do you think of "discussion" in place of article? It's not really an article, it's part of the documentation.

Reply:
What do you think about this? 

```suggestion
This section uses PGD cluster node names to refer to servers in the cluster. To get a list of node names in your cluster, use:
```